docker-compose.override.yaml

```diff
@@ -65,6 +65,17 @@ services:
         condition: service_started
       django:
         condition: service_healthy
+  # scoring database
+  scoredb:
+    image: postgis/postgis:latest
+    environment:
+      POSTGRES_USER: scoring
+      POSTGRES_DB: scoring
+      POSTGRES_PASSWORD: secretkey
+    volumes:
+      - postgresql-scoring-data:/var/lib/postgresql/data
+    ports:
+      - 5433:5432
 
```

Comment:
Can this be moved to `docker-compose.yaml`? In the current setup, if one is running just `docker-compose.yaml` for postgres/rabbit/minio etc, and running django/celery natively, then there is no way to run the scoring DB.

Reply:
yeah, I moved the other container into the main docker-compose.